### PR TITLE
[SPARK-29975][SQL][followup] document --CONFIG_DIM

### PR DESCRIPTION
**Pull request by cloud-fan**
### What changes were proposed in this pull request?

add document to address https://github.com/apache/spark/pull/26612#discussion_r349844327

### Why are the changes needed?

help people understand how to use --CONFIG_DIM

### Does this PR introduce any user-facing change?

no

### How was this patch tested?

N/A

### CODE REVIEW

**Comment by cloud-fan** [comment]
cc @maropu @HyukjinKwon

File: sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala [hunk 1]
```diff
@@ -304,7 +313,7 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
   protected def runQueries(
       queries: Seq[String],
       testCase: TestCase,
-      configSet: Option[Seq[(String, String)]]): Unit = {
+      configSet: Seq[(String, String)]): Unit = {
```
Comment (cloud-fan) [inline sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala]: not very related but to simplify things.

Comment (dongjoon-hyun) [reply sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala]: +1 for this simplication.

Comment (maropu) [reply sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala]: +1

**Review by maropu — APPROVED** [review]
LGTM, thanks!

**Review by HyukjinKwon — APPROVED** [review]

**Comment by maropu** [comment]
Merged to master. 
(Sorry, but I made a mistake to merge this before passing the tests. If any issue happens, I'll do somethig then.)

**Comment by maropu** [comment]
(I got relieved...)

**Pull request closed without merging**